forumhelpers/FHP.js

```diff
@@ -1,53 +1,60 @@
-var man_addition=""
-var man_pull = new XMLHttpRequest()
+function getData(dataSet) {
+	var grabDelay = 0
+	fetch("https://theforumhelpers.github.io/forumhelpers/"+dataSet+".json")
+		.then(response => response.json())
+		.then(data => {
+			document.getElementById(dataSet+"Number").innerHTML = data.length;
+			for (j = 0; j < data.length; j++) {
+				var name = data[j].name;
+				var id = data[j].id;
+				var bio = data[j].bio;
+				var section = document.getElementById(dataSet+"List").innerHTML;
+				var addition = `
+				<div class="profileContainer" id="${name}">
+					<h4 class="profileName"><a href="https://scratch.mit.edu/users/${name}/">${name}</a><span id="${name}Posts"></span></h4>
+					<div class="profileInner">
+						<a href="https://scratch.mit.edu/users/${name}/"><img src="https://cdn2.scratch.mit.edu/get_image/user/${id}_60x60.png" class="profilePicture" id="${name}pfp" loading="lazy" alt="${name}'s Profile Picture"></a>
+						<p class="profileBio">${bio}</p>
+					</div>
+					<p class="FullOcular" id="${name}Ocular" style="border: 5px solid black; border-radius: 10px; padding: 5px; margin-right:20%; width: fit-content" title="Ocular Status"><span class="ocularStatus" id="${name}Status">Loading Status...</span></p>
```

Comment:
Could you use External CSS instead of inline?

Reply:
Right, I was so intent on getting the CSS for that to look right that I forgot to move it to the stylesheet.